code/planning/src/planning/preplanner.py

```diff
@@ -93,39 +94,14 @@ def global_route_callback(self, data: CarlaRoute) -> None:
 
         self.global_route_backup = None
         # wrong start coordinates from  -> PR 191 solves it
-        x_start_ = self.agent_pos.x
-        y_start_ = self.agent_pos.y
+#        x_start_ = self.agent_pos.x
+#        y_start_ = self.agent_pos.y
         # after PR 191 is merged it should work without the manual position
         x_start = 983.5
-        y_start = -5373.2
-        # z_start = self.agent_pos.z
-        """ roll_start, pitch_start, yaw_start = \
-            tf.transformations.euler_from_quaternion(
-             [self.agent_ori.x, self.agent_ori.y,
-              self.agent_ori.z, self.agent_ori.w]) """
-        self.loginfo(f"x_start = {x_start_}")
-        self.loginfo(f"y_start = {y_start_}")
-        self.loginfo(f"x_start = {x_start}")
-        self.loginfo(f"y_start = {y_start}")
+        y_start = -5433.2
```

Comment:
cause even though it should be working its not yet and would cause the preplanner to fail in the middle of planning even though this is just a simple start point. 

@erlbacsi you might wanna take another look at this if possible cause the error occurs in a part of the big preplanning_trajectory.py file.

Reply:
In the dev environment the start position of the car is on the left side of the road. I chose a manual point on the right side of the road because for the calculation the car needs to be on the side where we are driving. Normally in the leaderboard scenarios i think we are on the correct side of the road. The point is only a workaround for the dev environment start position.
If we are on the left side -> trajectory is calculated on the left side
if we are on the rigth side -> trajectory is calculated on the right side